php/FlatbufferBuilder.php

```diff
@@ -593,6 +593,10 @@ public function endVector()
 
     protected function is_utf8($bytes)
     {
+        if (function_exists('mb_detect_encoding') && (bool) mb_detect_encoding($bytes, 'UTF-8', true)) {
```

Comment:
I guess you can temporarily just comment out the `if` to test the old code, that should have already been working.